packages/contracts/contracts/LPRewards/Dependencies/SafeERC20.sol

```diff
@@ -1,75 +0,0 @@
-// SPDX-License-Identifier: MIT
-
```

Comment:
🎗️  This one feels like it might be useful, even just to remember it existed at one point in case we need it

Reply:
it might come back when TBTC gets introduced